spec.bs

```diff
@@ -38,6 +38,8 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
     text: height; url: embedded-content-other.html#attr-dim-height
 spec: fetch; urlPrefix: https://fetch.spec.whatwg.org/
   type: dfn
+    text: request; url: #concept-request
```

Comment:
This concept is exported by fetch, so we shouldn't need this: https://github.com/whatwg/fetch/blob/main/fetch.bs#L1322